async-publish-actions.php

```diff
@@ -61,6 +61,13 @@ function _wpcom_do_async_transition_post_status( $post_id, $new_status, $old_sta
 		return;
 	}
 
+	// Gauge how often this feature is used.
+	if ( true === WPCOM_IS_VIP_ENV && false === WPCOM_SANDBOXED ) {
+		\Automattic\VIP\Stats\send_pixel( [
+			'vip-go-async-publish-job' => FILES_CLIENT_SITE_ID,
```

Comment:
Minor note - this handles all post status transitions (not just publish) so maybe we want the stat name to reflect that? `vip-go-async-post-transition-status-job` or similar?

Reply:
TIL that technically the job is only registered when going to/from publish: https://github.com/Automattic/vip-go-mu-plugins/blob/12f85b7e170425014854a1b1b1799871a5654f06/async-publish-actions.php#L151

But definitely not oppose to renaming the stat pixel.

Reply:
Oh i didn't realize that - alright in that case this is fine then.